src/components/ebay-fake-menu/index.marko

```diff
@@ -53,6 +54,8 @@ $ var baseClass = input.classPrefix || "fake-menu";
                         ...processHtmlAttributes(item, itemIgnoredAttributes)
                         class=[`${baseClass}__item`, item.class]
                         style=item.style
+                        disabled=(item.type === "button" && item.disabled)
+                        aria-disabled=(item.type !== "button" && item.disabled ? true : null)
```

Comment:
I think you need quotes around the `true` as it is only rendering `aria-disabled`. It should render `aria-disabled="true"`

Reply:
that's right; the `true` or `'true'` always gets me mixed up